NBitcoin/BitcoinCore/Coins.cs

```diff
@@ -9,29 +9,10 @@ namespace NBitcoin.BitcoinCore
 	public class Coins : IBitcoinSerializable
 	{
 		// whether transaction is a coinbase
-		bool fCoinBase;
-		public bool Coinbase
-		{
-			get
-			{
-				return fCoinBase;
-			}
-			set
-			{
-				fCoinBase = value;
-			}
-		}
+		public bool CoinBase { get; private set; }

```

Comment:
In that particular case CoinBase was assigned within the class, so I decided to not expose it.